### PR TITLE
refactor link wallet ui

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request refactors the linking wallet step for onboarding and also removes unnecessary buttons to navigate through the flow.

### Approvals

- [x] Dev
- [ ] Product

### CODE REVIEW

File: centrifuge-app/src/pages/Onboarding/BusinessOwnership.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const businessOwnershipInput = object({
   ),
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Onboarding/BusinessOwnership.tsx:34]: same here, the component will automatically trigger `nextStep` upon success now so no need to display a success message.

File: centrifuge-app/src/pages/Onboarding/InvestorType.tsx [hunk 2]
```diff
@@ -9,91 +9,51 @@ const examplePool = {
...
 
-export const InvestorType = ({
-  investorType,
-  isAgreedToDataSharingAgreement,
-  nextStep,
-  setInvestorType,
-  setIsAgreedToDataSharingAgreement,
-}: Props) => {
-  const [isDataSharingAgreementDialogOpen, setIsDataSharingAgreementDialogOpen] = useState(false)
+const InvestorTypeCard = styled(Card)`
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Onboarding/InvestorType.tsx:16]: abstracts the `Card` component below to a `styled-component`

File: centrifuge-app/src/pages/Onboarding/LinkWallet.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
-import { Box, Button, IconCheck, Shelf, Stack, Text } from '@centrifuge/fabric'
```
Comment (jpangelle) [inline centrifuge-app/src/pages/Onboarding/LinkWallet.tsx:1]: Refactors the linking wallet step to include the data sharing agreement and to only have one button like below
image

File: onboarding-api/src/index.ts [hunk 4]
```diff
@@ -6,6 +6,7 @@ const centrifugeDomains = [
   /^(https:\/\/.*cntrfg\.com)/,
   /^(https:\/\/.*centrifuge\.io)/,
   /^(https:\/\/.*altair\.network)/,
+  /^(https:\/\/pr-\d*--dev-app-cntrfg.netlify\.app)/,
```
Comment (jpangelle) [inline onboarding-api/src/index.ts:9]: adds domain for pr previews

**Review by sophialittlejohn — APPROVED** [review]
💯

**Comment by jpangelle** [comment]
> 🚀 Deployed on https://pr-1124--dev-app-cntrfg.netlify.app

for u @annamehr

**Pull request merged**